### DISCUSSION TITLE
Add a new article about surface masks (also perhaps add ability to reset them).

### DISCUSSION THREAD

**Discussion by Jdance-Media** [SmartlyDressedGames/Unturned-3.x-Community#4071] [Ideas]
Hi. For the past few months I've been seeing an uptick in those having issues with the surface mask setting for the map editor. Of course, basic users without any background knowledge on raymask layers and bitwise integers should not be using it, but when a basic user does change the setting, they manage to make some tools unusable.

It may help us out if there was some kind of support article explaining a little bit about what the setting is. Alongside that, if the article could provide the default values, users could once again use tools they accidentally messed up. Referring to https://support.smartlydressedgames.com

In addition, an idea for in-game, is to have right-clicking buttons/inputs reset their values.

**Comment by SDGNelson** [comment]
You're right. The surface properties field is a mess right now. There are support tickets for it multiple times per week. I need to recreate the bitmask field for the regular editor. 

I've now posted an article on the support site: https://support.smartlydressedgames.com/hc/en-us/articles/18346795651604.

**Reply by Jdance-Media** [reply]
Thanks. This will hopefully help with the support posts. 👍
